### PR TITLE
Add links to Redocly for sign-in, sign-out, and error pages.

### PR DESCRIPTION
**Pull request by gormanho-okta**
## Description:
- **What's changed?** Add links to Redocly for sign-in, sign-out, and error pages. Also adds the Redocly sitemap to Vuepress's sitemap index.
- **Is this PR related to a Monolith release?** Yes, ping @gormanho-okta to see if Redocly changes will go into Jan monthly

### Resolves:

* [OKTA-508039](https://oktainc.atlassian.net/browse/OKTA-508039)

### CODE REVIEW

**Review by royalchan-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 1]
```diff
@@ -2261,6 +2261,18 @@ HTTP/1.1 200 OK
 }
 ```
 
+## Sign-in page operations
+
+Please see the [sign-in page API documentation](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/getSignInPage) on the new beta reference site.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2266]:
```suggestion
See the [Customized Sign-in Page API reference](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/getSignInPage) on the new beta reference site.
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 2]
```diff
@@ -2261,6 +2261,18 @@ HTTP/1.1 200 OK
...
+
+## Sign-out page operations
+
+Please see the [sign-out page API documentation](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/getSignOutPageSettings) on the new beta reference site.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2270]:
```suggestion
See the [Customized Sign-out Page API reference](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/getSignOutPageSettings) on the new beta reference site.
```

File: packages/@okta/vuepress-site/docs/reference/api/brands/index.md [hunk 3]
```diff
@@ -2261,6 +2261,18 @@ HTTP/1.1 200 OK
...
+
+## Error page operations
+
+Please see the [error page API documentation](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/getErrorPage) on the new beta reference site.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/brands/index.md:2274]:
```suggestion
See the [Customized Error Page API reference](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/getErrorPage) on the new beta reference site.
```

File: packages/@okta/vuepress-site/docs/reference/beta-only/index.md [hunk 4]
```diff
@@ -8,4 +8,7 @@ We've got a new API reference in the works! With a fresh look and feel, our new
 
 In some cases, APIs have only been documented on the new beta reference site, and not on the old API reference that you are currently looking at. This page provides a list of those, so that you can easily find them.
 
-* [Agent Pools](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/AgentPools/#tag/AgentPools)
\ No newline at end of file
+* [Agent Pools](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/AgentPools/#tag/AgentPools)
+* [Sign-in Page](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/getSignInPage)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/beta-only/index.md:12]:
```suggestion
* [Customized Sign-in Page](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/getSignInPage)
```

File: packages/@okta/vuepress-site/docs/reference/beta-only/index.md [hunk 5]
```diff
@@ -8,4 +8,7 @@ We've got a new API reference in the works! With a fresh look and feel, our new
...
+* [Sign-out Page](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/getSignOutPageSettings)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/beta-only/index.md:13]:
```suggestion
* [Customized Sign-out Page](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/getSignOutPageSettings)
```

File: packages/@okta/vuepress-site/docs/reference/beta-only/index.md [hunk 6]
```diff
@@ -8,4 +8,7 @@ We've got a new API reference in the works! With a fresh look and feel, our new
...
+* [Error Page](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/getErrorPage)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/reference/beta-only/index.md:14]:
```suggestion
* [Customized Error Page](https://developer.okta.com/docs/api/openapi/okta-management/management/tag/Customization/#tag/Customization/operation/getErrorPage)
```

**Review by vanngo-okta — COMMENTED** [review]
A few style nits. Should we add "Customized" before the API name to align with the "Customizations" tag in redocly?

**Comment by vanngo-okta** [comment]
Hi @gormanho-okta, the content in `packages/@okta/vuepress-site/docs/reference/beta-only/index.md` is duplicated in `https://developer.okta.com/docs/reference/#apis-documented-only-on-the-new-beta-reference`. We are thinking of removing the `/reference/beta-only/index.md` page and only keeping the `/reference/#apis-documented-only-on-the-new-beta-reference` content to avoid duplication (and double entry). Can you please check if this link (https://developer.okta.com/docs/reference/#apis-documented-only-on-the-new-beta-reference) can be referenced from Redocly? Thanks!

**Review by vanngo-okta — APPROVED** [review]
LGTM!

**Pull request merged**